### PR TITLE
Added detail about environment configuration loading and new added module config location

### PR DESCRIPTION
**Pull request by iturgeon**
Document updates for related pull request https://github.com/fuel/core/pull/1106 & commit https://github.com/iturgeon/core/commit/4113c5e23be3fb76e80af6ef1b26930f51c824d9
for adding loading of environment specific configurations for modules in to the main app config directory.

Signed-off-by: Ian Turgeon iturgeon@gmail.com

### CODE REVIEW

**Comment by WanWizard** [comment]
Config::get() doesn't load anything. And if you mean load(), you have to drop the 'mypackage' prefix.

File: general/environments.html [hunk 1]
```diff
@@ -123,6 +123,41 @@ <h3 id="env_config">Environments and Config</h3>
...
+ * Locating a package config
+ * 
+ * Loading Order assuming FUEL::TEST:
+ * 
+ * 1. app/packages/mypackage/config/stuff.php
+ * 2. app/config/stuff.php
+ * 3. app/packages/mypackage/config/test/stuff.php *note below
+ * 4. app/config/test/stuff.php
+ */
+Config::get('mypackage.stuff');</code></pre>
```
Comment (frankdejonge) [inline general/environments.html:143]: You've got Config::load and Config::get mixed up here.

File: general/environments.html [hunk 2]
```diff
@@ -123,6 +123,41 @@ <h3 id="env_config">Environments and Config</h3>
...
+			<pre class="php"><code>/**
+ * Locating a module config
+ * 
+ * Loading Order assuming FUEL::TEST:
+ * 
+ * 1. app/modules/mymodule/config/stuff.php
+ * 2. app/modules/mymodule/config/test/stuff.php
+ * 3. app/config/test/modules/mymodule/stuff.php
+ */
+Config::get('mymodule::stuff');</code></pre>
```
Comment (frankdejonge) [inline general/environments.html:158]: Same here

**Comment by iturgeon** [comment]
The associated pull request has been abandoned, but there still may be some use to outlining the order in which these configurations get loaded.

Is there any value in fixing up this pull request or should it be abandoned?

**Comment by WanWizard** [comment]
I never got an answer to my last question on that pull request, so i guess the answer is yours to give.

**Comment by WanWizard** [comment]
No response anymore, and since this is a pull request against 1.3, I close it.

**Pull request closed without merging**